src/pr-info.ts

```diff
@@ -385,26 +385,30 @@ const configSuspicious = <ConfigSuspicious>(async (path, getContents) => {
     if (text === undefined) return `couldn't fetch contents`;
     const tester = configSuspicious[basename];
     let suspect: string | undefined;
-    if (tester.length === 1) {
+    if (tester.length <= 1) {
```

Comment:
Why is this change needed?

Reply:
This PR depends on #229 and this change is part of that PR. Can we land #229? I think it's ready ... If so I'll rebase this PR and remove its "draft" status. That'll obviate the first commit including this change.

Reply:
OK, merged that (I didn't because of the PR it is tied too, but this one makes it better).

Anyway, can you get it back to the `===` now?

Reply:
> OK, merged that (I didn't because of the PR it is tied too, but this one makes it better).

Thanks! Rebased.

> Anyway, can you get it back to the `===` now?

Can we remove it entirely, now that all the checkers use both arguments?

Reply:
👍